Scripts to each paragon/boomerang monkey script.ahk

```diff
@@ -4,7 +4,7 @@ SendMode Input              ; Recommended for new scripts due to its superior sp
 SetWorkingDir %A_ScriptDir% ; Ensures a consistent starting directory.
 
 ; This function is used go x times to the right on the map selection menu
-ToPage(number_of_times) {
+ToPage(0) {

```

Comment:
This is not the correct way to adjust which page you go to.

As you can see down in line 27. You should just uncomment be removing ";" and then write the page number

Like if you want to go to the first page its: ToPage(1) - Starting from 1 and not 0.

Reply:
Then you can simply update the following MouseClick on line 31 or add a newline with the coordinates to another map.